src/Type/Php/VersionCompareFunctionDynamicReturnTypeExtension.php

```diff
@@ -77,4 +85,24 @@ public function getTypeFromFunctionCall(
 		return TypeCombinator::union(...$types);
 	}
 
+	/**
+	 * @return ConstantStringType[]
+	 */
+	private function getVersionStrings(Expr $expr, Scope $scope): array
+	{
+		if (
+			$expr instanceof Expr\ConstFetch
```

Comment:
[PHP_EXTRA_VERSION](https://www.php.net/manual/en/reserved.constants.php#constant.php-extra-version) should be probably support as well.